spec/lib/html_scrubber_util_spec.rb

```diff
@@ -42,11 +42,38 @@ class TestingScrubber
   end
 
   context "given a style attribute" do
-    let(:source_html) { "<div style=\"somewhere\"></div>" }
+    let(:source_html) { "<style>Test</style>" }
```

Comment:
Why the change here?  This contradicts the context's description: "given a style attribute".  We should keep the descriptions aligned with what's being tested.

Reply:
Had talked off GitHub about this with Tristan; The previous PR to add this test accidentally duplicated the test (https://github.com/ideacrew/enroll/blob/trunk/spec/lib/html_scrubber_util_spec.rb#L35-L51) when one of them should'vebeen testing the style tag vs the style attribute. Maybe the only other thing is to change the context of this test to say "given a style tag"

Reply:
@ATBull81 if I'm understanding correctly, we also want to include style tags? Going to update description

Reply:
Updated! Let me know if the new change works